src/PlusDataCollection/OptiTrack/vtkPlusOptiTrack.cxx

```diff
@@ -378,7 +378,10 @@ void vtkPlusOptiTrack::vtkInternal::InternalCallback(sFrameOfMocapData* data, vo
     rigidBodyToTrackerMatrix->Identity();
     sRigidBodyData currentRigidBody = rigidBodies[rigidBodyId];
 
-    if (currentRigidBody.MeanError != 0)
+    // Check if tool is in view
+    bool bTrackingValid = currentRigidBody.params & 0x01; 
+
+    if ((currentRigidBody.MeanError != 0) & (bTrackingValid))
```

Comment:
This changes the behaviour to only send valid transforms. Plus supports the ability to send invalid transforms with the status flag set. If you want to pass this information to the client, you'd need to modify line 406.

Reply:
I'm not sure. Invalid transforms are sent through line 412. This is the whole point of this PR. 
I think ```currentRigidBody.MeanError != 0``` is supposed to detect if a marker is visible (?), but is always true, even when markers are not visible (tested with Optitrack Duo camera and SendValidTransformsOnly="FALSE"). I left it in case there is another purpose of checking meanError != 0?